src/lib/layout/navigation.svelte

```diff
@@ -2,81 +2,152 @@
     import { base } from '$app/paths';
     import { page } from '$app/stores';
     import { trackEvent } from '$lib/actions/analytics';
+    import { tooltip } from '$lib/actions/tooltip';
+    import { slide } from '$lib/helpers/transition';
 
     $: project = $page.params.project;
-    $: path = `${base}/console/project-${project}`;
+    $: projectPath = `${base}/console/project-${project}`;
+
+    $: secondSideNav = $page.data.secondSideNav;
+    // We need to have this second variable, because we only want narrow
+    // to change automatically if we change from having a second side nav to
+    // not having one, not when the second side nav changes to a different value.
+    $: hasSecondSideNav = !!secondSideNav;
```

Comment:
can we maybe give it a clearer naming? 🤔 

I think something like subNavigation or whatever would work better